translation/es/patterns/30-day-warranty.md

```diff
@@ -0,0 +1,76 @@
+## Title
+
+Garantía de 30 Días
+
+## Patlet
+
+Al aceptar contribuciones de fuera de tu propio equipo, existe una aversión natural a asumir la responsabilidad del código no escrito por el equipo mismo. A través de la Garantía de 30 Días, el equipo contribuyente se compromete a proporcionar correcciones de errores al equipo receptor, lo que aumentará el nivel de confianza entre ambos equipos y hace más probable que se acepten las contribuciones.
+
+## Problema
+
+Un equipo desarrolla un componente que se utiliza en toda una organización. Este equipo se resiste a aceptar o directamente rechaza contribuciones (solicitudes de funcionalidades). Este comportamiento bloquea el progreso y conduce a interrupciones frecuentes por escalamientos.
+
+## Contexto
+
+- Los equipos dependen de que otro equipo acepte sus contribuciones para que un componente producido por el equipo receptor pueda ser utilizado por el equipo contribuyente.
+- El equipo receptor no tiene los recursos, conocimientos, permisos y/o inclinación para escribir el componente/funcionalidad contribuido por sí mismos.
+
+## Fuerzas
```

Comment:
Thinking to change the word "Fuerzas" by "Resistencias".

In the context of the description of [Forces](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/meta/pattern-template.md#forces) the concept of "Resistencias" (could be translated as constraints or resistances) could fit better for all the patterns. This comment is open for the structure of any pattern. "Fuerzas" is just the literal translation ... but this is my personal opinion. 

What do you think @ovas04 ? Open also for any other Spanish native speaker.

Reply:
@ovas04 for now, let's only make the suggested changes for Spanish.

If we want to make changes for pt-br as well, we can change that later in a separate PR.